specification/documentation/GetMessage.md

```diff
@@ -34,6 +35,17 @@ Each channel can have one of the following statuses:
 * `delivered` - the channel has delivered the message
 * `failed` - the channel has failed to deliver the message
 
+Each channel can have one of the following supplier statuses:
```

Comment:
We probably want to be explicit in saying these supplier statuses are specific to the NHSAPP channel

Reply:
Originally I think we said we'd have a section specifically for `Supplier Statuses` like we have for `Message Statuses` further down